### PR TITLE
change ASG to ASC to enforce set styling standards

### PR DESCRIPTION
**Pull request by sr229**
### Description

Changes the name of the folder ``Article_Style_Guide`` to ``Article_Styling_Criteria`` and rephrases ``en.md``. Locales and news are unaffected

### Status
Finished; Pending Review

### Changelog
- ASG > ASC Rename
- Redo some sentences in ``en.md``

### Related Issues 

resolves ppy/osu-wiki#535

### CODE REVIEW

File: wiki/Article_Styling_Criteria/en.md [hunk 1]
```diff
@@ -1,581 +1,559 @@
...
+### Point of view
+
+When referring to the player, use `the player` or `a player`.
+
+**Never** use `he`/`his` or `she`/`her`! Instead, use `they`/`their`/`theirs`.
+
+In instances where you are referring directly to the reader (e.g. tutorial page), you could use the word `you`.
+
+
+**Never** use the first person perspective, `I`, or `I am`/`I'm`.
```
Comment (MegaApplePi) [inline wiki/Article_Styling_Criteria/en.md]: Why did you remove `### Terminology`? It literally is just a section that shows how certain words are to be spelt.

Comment (MegaApplePi) [inline wiki/Article_Styling_Criteria/en.md]: Just `I`, you're overkilling it with more "I" examples.

File: wiki/Article_Styling_Criteria/en.md [hunk 2]
```diff
@@ -1,581 +1,559 @@
...
----
-
-Use "beatmaps" instead of "maps".
-
-Use "creator" instead of "beatmapper" or "mapper".
+# Article Styling Criteria
+
+_See also: [Article Styling Critria/News](/wiki/Article_Styling_Criteria/News)._
+
+The article styling crtieria serves as a enforced styling standard to keep consistency in clarity, formatting, and layout in all articles of the osu!wiki.
```
Comment (MegaApplePi) [inline wiki/Article_Styling_Criteria/en.md]: `The article styling crtieria serves ...` ->
`The article styling criteria (ASC) serves ...`
- add `(ASC)`
- typo `crtieria` -> `criteria`

File: wiki/Article_Styling_Criteria/en.md (hunk 1 above)
Comment (sr229) [reply wiki/Article_Styling_Criteria/en.md]: being verbose is the key 👍

Comment (sr229) [reply wiki/Article_Styling_Criteria/en.md]: probably the rebase fucked it up, I'll recover that later

**Comment by sr229** [comment]
@MegaApplePi thanks for spotting the issues, all fixed now
